app/activeproject.cpp

```diff
@@ -492,22 +488,26 @@ void ActiveProject::updateActiveLayer()
 {
   if ( !layerVisible( mActiveLayer.layer() ) )
```

Comment:
You moved the function to `InputUtils`, let's use it and remove the one defined here.

```suggestion
  if ( !InputUtils::layerVisible( mActiveLayer.layer() ) )
```